image/image.go

```diff
@@ -29,30 +29,24 @@ func IDFromDigest(digest digest.Digest) ID {
 
 // V1Image stores the V1 image configuration.
 type V1Image struct {
-	// ID a unique 64 character identifier of the image
-	ID string `json:"id,omitempty"`
-	// Parent id of the image
-	Parent string `json:"parent,omitempty"`
-	// Comment user added comment
-	Comment string `json:"comment,omitempty"`
-	// Created timestamp when image was created
-	Created time.Time `json:"created"`
-	// Container is the id of the container used to commit
-	Container string `json:"container,omitempty"`
+	ID        string    `json:"id,omitempty"`        // ID is a unique 64 character identifier of the image
+	Parent    string    `json:"parent,omitempty"`    // Parent id of the image
+	Comment   string    `json:"comment,omitempty"`   // Comment is user added comment
+	Created   time.Time `json:"created"`             // Created timestamp when image was created
+	Container string    `json:"container,omitempty"` // Container is the id of the container used to commit
+
 	// ContainerConfig is the configuration of the container that is committed into the image
 	ContainerConfig container.Config `json:"container_config,omitempty"`
-	// DockerVersion specifies version on which image is built
-	DockerVersion string `json:"docker_version,omitempty"`
-	// Author of the image
-	Author string `json:"author,omitempty"`
+
+	DockerVersion string `json:"docker_version,omitempty"` // DockerVersion specifies version on which image is built
+	Author        string `json:"author,omitempty"`         // Author of the image
+
 	// Config is the configuration of the container received from the client
 	Config *container.Config `json:"config,omitempty"`
-	// Architecture is the hardware that the image is build and runs on
-	Architecture string `json:"architecture,omitempty"`
-	// OS is the operating system used to build and run the image
-	OS string `json:"os,omitempty"`
-	// Size is the total size of the image including all layers it is composed of
-	Size int64 `json:",omitempty"`
+
+	Architecture string `json:"architecture,omitempty"` // Architecture is the hardware that the image is build and runs on
```

Comment:
No, I meant that with this change, we mix formatting to put the description _above_ some, and _to the right_ of others;

<img width="973" alt="screen shot 2016-11-29 at 11 00 42" src="https://cloud.githubusercontent.com/assets/1804568/20705458/4d91be1e-b623-11e6-9f32-47396dfa73a2.png">

I would definitely prefer to have that in a separate PR if you think it's important (and why).

Even though I can see some formatting being slightly more readable, we should also take into account that docker gets a lot of pull requests; formatting should be such that it minimizes the chance of merge conflicts.